### ISSUE TITLE
Can't install under 0.6.4

### ISSUE THREAD

**Issue by arbenede** [JuliaPolyhedra/QHull.jl#21]
I am getting this error with Julia 0.6.4 under Windows 10:

```julia
julia> Pkg.add("QHull")
ERROR: unknown package QHull
macro expansion at .\pkg\entry.jl:53 [inlined]
(::Base.Pkg.Entry.##1#3{String,Base.Pkg.Types.VersionSet})() at .\task.jl:335
Stacktrace:
 [1] sync_end() at .\task.jl:287
 [2] macro expansion at .\task.jl:303 [inlined]
 [3] add(::String, ::Base.Pkg.Types.VersionSet) at .\pkg\entry.jl:51
 [4] (::Base.Pkg.Dir.##4#7{Array{Any,1},Base.Pkg.Entry.#add,Tuple{String}})() at .\pkg\dir.jl:36
 [5] cd(::Base.Pkg.Dir.##4#7{Array{Any,1},Base.Pkg.Entry.#add,Tuple{String}}, ::String) at .\file.jl:59
 [6] #cd#1(::Array{Any,1}, ::Function, ::Function, ::String, ::Vararg{String,N} where N) at .\pkg\dir.jl:36
```

Any ideas?

**Comment by blegat** [comment]
It has not been released yet, you should do `Pkg.clone("https://github.com/JuliaPolyhedra/QHull.jl.git")`

**Comment by blegat** [comment]
It is now released under Julia v1.0: https://github.com/JuliaLang/METADATA.jl/pull/19111
You can install it with
```julia
] add QHull
```
